tst/EnergyPlus/unit/HeatBalanceSurfaceManager.unit.cc

```diff
@@ -3126,4 +3127,355 @@ TEST_F(EnergyPlusFixture, HeatBalanceSurfaceManager_TestInitHBInterzoneWindow)
     InitIntSolarDistribution(*state);
     EXPECT_NEAR(1.666667, state->dataHeatBal->SurfIntBmIncInsSurfIntensRep(1), 0.00001);
 }
+
+TEST_F(EnergyPlusFixture, HeatBalanceSurfaceManager_QdotConvInRepAdjRatioTest)
+{
+    bool ErrorsFound(false);
+
+    std::string const idf_objects =
+        delimited_string({"Material,",
+                          "  Concrete Block,          !- Name",
+                          "  MediumRough,             !- Roughness",
+                          "  0.1014984,               !- Thickness {m}",
+                          "  0.3805070,               !- Conductivity {W/m-K}",
+                          "  608.7016,                !- Density {kg/m3}",
+                          "  836.8000;                !- Specific Heat {J/kg-K}",
+                          "Construction,",
+                          "  WallConstruction,        !- Name",
+                          "  Concrete Block;          !- Outside Layer",
+                          "WindowMaterial:SimpleGlazingSystem,",
+                          "  WindowMaterial,          !- Name",
+                          "  5.778,                   !- U-Factor {W/m2-K}",
+                          "  0.819,                   !- Solar Heat Gain Coefficient",
+                          "  0.881;                   !- Visible Transmittance",
+                          "Construction,",
+                          "  WindowConstruction,      !- Name",
+                          "  WindowMaterial;          !- Outside Layer",
+                          "WindowProperty:FrameAndDivider,",
```

Comment:
No need for a WindowProperty:FrameAndDivider

Reply:
Thanks, removed it.